dev/com.ibm.ws.security.authorization.jacc.ejb/.classpath

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="test"/>
```

Comment:
Add output="bin_test" to this classpathentry.

Reply:
Done.